src/main/java/org/openpnp/machine/reference/vision/ReferenceFiducialLocator.java

```diff
@@ -177,40 +183,49 @@ public Location getHomeFiducialLocation(Location location, Part part) throws Exc
      * @return
      * @throws Exception
      */
-    private static Location getFiducialLocation(BoardLocation boardLocation, Placement fid)
-            throws Exception {
+	private static Location getFiducialLocation(BoardLocation boardLocation, Placement fid, Length fidDia) throws Exception {
         Camera camera = Configuration.get().getMachine().getDefaultHead().getDefaultCamera();
 
         Logger.debug("Locating {}", fid.getId());
 
+		Footprint footprint;
+
+		// Careful...magic number below. Also used in JobPanel. This are for
+		// global panel fiducials, which do not have an entry in the PCB BOM.
+		if ((fid.getId().equals("PanelFid1_a322") || fid.getId().equals("PanelFid2_g301"))) {
```

Comment:
Right, so the interface goes from:
```
public Location locateBoard(BoardLocation boardLocation) throws Exception;
```
to 
```
public Location locateBoard(BoardLocation boardLocation, boolean checkPanel) throws Exception;
```

and in the implementation, if that boolean is true, we just check panel fids instead of board fids.

Another option is to just duplicate that method in the interface, add the parameter and in the implementations default to false. This would mean that no existing code has to change. You would just add the additional call into your button and to the ReferenceJobProcessor at the start of the job. This may be a better option.

So, the interface becomes:
```
public interface FiducialLocator extends PropertySheetHolder {
    public Location locateBoard(BoardLocation boardLocation) throws Exception;
    public Location locateBoard(BoardLocation boardLocation, boolean checkPanel) throws Exception;
    public Location getHomeFiducialLocation(Location location, Part part ) throws Exception;
}
```

And in ReferenceFiducialLocator:
```
    public Location locateBoard(BoardLocation boardLocation) throws Exception {
        return locateBoard(boardLocation, false);
    }

    public Location locateBoard(BoardLocation boardLocation, boolean checkPanel) throws Exception {
        // Find the fids in the board
        IdentifiableList<Placement> fiducials = getFiducials(boardLocation);
...
```